internal/distro/rhel85/distro.go

```diff
@@ -0,0 +1,168 @@
+package rhel85
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
+	"github.com/osbuild/osbuild-composer/internal/distro"
+	"github.com/osbuild/osbuild-composer/internal/rpmmd"
+)
+
+const name = "rhel-85"
```

Comment:
I think we decided to change the name to `rhel-8.5`, right? Should we do it straight away?

Reply:
Just a note, that this will affect Weldr and other parts, which rely on the name to be without a dot. Something to consider, but I agree that it would make sense to start using the dot.

Reply:
I started the work with the new naming scheme on the branch and realised that the distro name detection would need to change, or we would need a condition special for 8.5.  Since the concept of aliases came up in that discussion (which is a good idea), I think we should do it in a separate PR where we add support for distro aliases and rename or alias existing distros.

Reply:
Makes sense!